src/helm/benchmark/presentation/run_specs_dec2023.conf

```diff
@@ -0,0 +1,169 @@
+  {description: "math:model=text_code,subject=intermediate_algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=prealgebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=geometry,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=counting_and_probability,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=precalculus,level=4,use_chain_of_thought=True", priority: 4}
+
+  {description: "math:model=text_code,subject=number_theory,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=prealgebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=geometry,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=counting_and_probability,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=precalculus,level=5,use_chain_of_thought=True", priority: 3}
+
+  # GSM
+  {description: "gsm:model=text_code", priority: 2}
+
+  # HumanEval
+  {description: "code:model=text_code,dataset=humaneval", priority: 1}
+
+  # LSAT
+  {description: "lsat_qa:model=text_code,task=all", priority: 2}
+  {description: "lsat_qa:model=text_code,task=grouping", priority: 3}
+  {description: "lsat_qa:model=text_code,task=ordering", priority: 3}
+  {description: "lsat_qa:model=text_code,task=assignment", priority: 3}
+  {description: "lsat_qa:model=text_code,task=miscellaneous", priority: 3}
+
+  # LegalSupport
+  {description: "legal_support:model=text_code", priority: 2}
+
+  # MedQA
+  {description: "med_qa:model=text_code", priority: 2}
+
+  # WMT
+  {description: "wmt_14:language_pair=en-cs,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-de,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-fr,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-hi,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-ru,model=text", priority: 3}
+  {description: "wmt_14:language_pair=cs-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=de-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=fr-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=hi-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=ru-en,model=text", priority: 2}
+
+  # RealToxicityPrompts
+  {description: "real_toxicity_prompts:model=text", priority: 2}
```

Comment:
I am ok keeping, but honestly I was never too convinced by RealToxicityPrompts even from when Suchin and co. made the dataset. But I guess none of the better approaches to toxicity became mainstream and I don't have a constructive alternative.

But honestly I might also just cut it.

*More fundamentally, this means the semantics of this are no longer exclusively about accuracy.*

Reply:
I'm conceptually okay with a leaderboard with both accuracy and non-accuracy metrics, but that would require some hacking to make this work with the existing summarize.py / schema.yaml system.

Reply:
I'm cutting.